README.md

```diff
@@ -2,11 +2,15 @@
 
 Skupper enables cloud communication by enabling you to create a Virtual Application Network.
 
-This upper layer network decouples addressing from the underlying network infrastructure, providing a secure and simple alternative to VPNs.
+This application layer network decouples addressing from the underlying network infrastructure.
+One example usage is to provide a secure and simple alternative to a VPN.
```

Comment:
Personally I prefer the notion that skupper enables secure communication across clusters 'without the need for a VPN' rather than being an 'alternative to a VPN'. I appreciate the difference there may be subtle (or only in my own head!), but I don't think we want to be seen as in the same category as a VPN. We want to allow people to communicate securely without needing VPNs.